Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -489,6 +523,22 @@ def get_alert_attachments(self, alert_id: int) -> dict[str, Any]:
         )
         return response_content
 
+    def get_compromised_credentials(self, alert_id: int) -> str:
```

Comment:
```suggestion
    def get_compromised_credentials(self, alert_id: str) -> str:
```
the alert_id came from args, and the args are str as long as they are not converted to another type

Reply:
ty